src/contract-deployments/mainnet-dao.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@bbenligiray Feel free to commit the deployed addresses here, these are wrong atm.

Reply:
I'll commit them.